bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala

```diff
@@ -31,20 +31,19 @@ object VoTableParser extends VoTableParser {
   val UCD_MAG        = UcdWord("phot.mag")
   val STAT_ERR       = UcdWord("stat.error")
 
-  val xsd = "/votable-1.2.xsd"
+  private def validate(catalogName: CatalogName, xmlText: String): CatalogProblem \/ Unit =
+    \/.fromTryCatchNonFatal {
+      import javax.xml.transform.stream.StreamSource
+      import javax.xml.validation.SchemaFactory
 
-  private def validate(xmlText: String): Throwable \/ String = \/.fromTryCatchNonFatal {
-    import javax.xml.transform.stream.StreamSource
-    import javax.xml.validation.SchemaFactory
+      val xsd        = s"/votable-${catalogName.voTableVersion.format}.xsd"
```

Comment:
The version should probably come from the XML itself, at some point.

Reply:
👍